frame/pablo/src/lib.rs

```diff
@@ -717,15 +709,22 @@ pub mod pallet {
 			let pool = Self::get_pool(pool_id)?;
 			let pool_account = Self::account_id(&pool_id);
 			let (added_base_amount, added_quote_amount, minted_lp) = match pool {
-				PoolConfiguration::StableSwap(info) => StableSwap::<T>::add_liquidity(
-					who,
-					info,
-					pool_account,
-					base_amount,
-					quote_amount,
-					min_mint_amount,
-					keep_alive,
-				)?,
+				PoolConfiguration::StableSwap(info) => {
+					let (added_base_amount, added_quote_amount, minted_lp, base_fee, quote_fee) =
+						StableSwap::<T>::add_liquidity(
+							who,
+							info.clone(),
+							pool_account.clone(),
+							base_amount,
+							quote_amount,
+							min_mint_amount,
+							keep_alive,
+						)?;
+					// imbalance fees
+					Self::disburse_fees(&pool_account, &info.owner, &base_fee)?;
```

Comment:
Suggestion: using `base_owner_fee` name or similar makes it easy to read that only owner fee is required to transfer, and other portion of the fee is already deposited into pool.

Reply:
May be you can clarify what you mean here? no fees are transferred before this step as of now.

Reply:
Oh I see. Makes sense now.